Bottom: 5ddaf23c0008b7c58867eb8dcdbf35ed23d8b46c
Top:    017592d95040e1620bd6e0ce23ec237e2cef38e8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-26 09:15:48 -0500

Refresh of tmp-wiki-database

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 29244da3..0a9243fb 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -110,8 +110,6 @@
       MYSQL_ROOT_PASSWORD: '{{ wiki_db_root_password }}'
       MYSQL_USER: '{{ wiki_db_user }}'
       MYSQL_PASSWORD: '{{ wiki_db_password }}'
-    networks:
-      - name: '{{ docker_network_frontend }}'
     state: started
 
 - name: wait for mysql initialization to complete
@@ -141,7 +139,7 @@
       - '{{ openldap_container_name }}:wiki_ldap'
       - '{{ wiki_db_container_name }}:wiki_db'
       - '{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
-    volumes: 
+    volumes:
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
       - '{{ wiki_dv_name }}/var/www/html:z'
     env:
